src/ServicePulse.Host/vue/src/components/monitoring/EndpointList.vue

```diff
@@ -140,22 +140,22 @@ function formatGraphDecimal(input, deci) {
               <div class="col-xs-2 col-xl-7 endpoint-name name-overview">
                 <div class="row box-header">
                   <div class="col-lg-max-3 no-side-padding lead righ-side-ellipsis endpoint-details-link">
-                    <a @click="navigateToEndpointDetails($event, endpoint.name)" class="cursorpointer" :title="`${endpoint.name}`">
+                    <a @click="navigateToEndpointDetails($event, endpoint.name)" class="cursorpointer" v-tooltip :title="endpoint.name">
                       {{ endpoint.name }}
                     </a>
                   </div>
-                  <span class="endpoint-count ng-binding ng-scope" v-if="endpoint.connectedCount || endpoint.disconnectedCount" :title="`Endpoint instance(s): ${endpoint.connectedCount || 0} `">({{ endpoint.connectedCount || 0 }})</span>
+                  <span class="endpoint-count ng-binding ng-scope" v-if="endpoint.connectedCount || endpoint.disconnectedCount" v-tooltip :title="`Endpoint instance(s):`+ endpoint.connectedCount || 0 ">({{ endpoint.connectedCount || 0 }})</span>
```

Comment:
Same as before, I suppose the plus sign is a workaround for something

Reply:
I am concatenating the value of "instance.erroCount " and a string text.